test/constraint/spec.test.ts

```diff
@@ -532,6 +532,39 @@ describe('constraints/spec', () => {
     });
   });
 
+  describe('omitAggregatePlotWithoutDimension', () => {
+    const CONFIG_WITH_OMIT_AGGREGATE_PLOT_WITHOUT_DIMENSION = extend({}, DEFAULT_QUERY_CONFIG, {omitAggregatePlotWithoutDimension: true});
+    it('should return false if plot is aggregate and has no dimension', () => {
+      const specM = buildSpecQueryModel({
+        mark: Mark.POINT,
+        encodings: [
+          {channel: Channel.X, field: 'Q', type: Type.QUANTITATIVE, aggregate: AggregateOp.MEAN}
+        ]
+      });
+      assert.isFalse(SPEC_CONSTRAINT_INDEX['omitAggregatePlotWithoutDimension'].satisfy(specM, schema, CONFIG_WITH_OMIT_AGGREGATE_PLOT_WITHOUT_DIMENSION));
+    });
+
+    it('should return true if plot is aggregate and has dimension', () => {
+      const specM = buildSpecQueryModel({
+        mark: Mark.POINT,
+        encodings: [
+          {channel: Channel.X, field: 'N', type: Type.NOMINAL, aggregate: AggregateOp.MEAN}
```

Comment:
This is cheating.  You can't aggregate nominal in practice.  Please re-write a proper test case with N and MEAN(Q)